src/pages/Main/AddMovieButton.jsx

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+
+function AddMovieButton(props) {
+  return (
+    <AddMovieButtonWrapper isDisplay={props.display}>
+      <AddMovieBtn onClick={props.onAddMovieListClick}>
+        <AddMovieButtonContent>더보기</AddMovieButtonContent>
+      </AddMovieBtn>
+    </AddMovieButtonWrapper>
+  );
+}
+const AddMovieButtonWrapper = styled.div`
+  display: ${props => (props.isDisplay ? 'block' : 'none')};
+  margin-top: 30px;
+  width: 1100px;
+`;
+const AddMovieBtn = styled.button`
+  width: 100%;
```

Comment:
```suggestion
}

const AddMovieButtonWrapper = styled.div`
  display: ${props => (props.isDisplay ? 'block' : 'none')};
  margin-top: 30px;
  width: 1100px;
`;

const AddMovieBtn = styled.button`
  width: 100%;
```
구분되는 요소들끼리는 띄어쓰기 해주세요

Reply:
넵 띄어쓰기 하겠습니다.